openshift/ansible/build-react.yaml

```diff
@@ -0,0 +1,17 @@
+---
+  - hosts: localhost
+    tasks:
+      - name: check pr exists and is open
+        uri:
+          url: https://api.github.com/repos/{{ github_owner }}/{{ github_repo }}/pulls/{{ PR }}
+          return_content: yes
+        register: pull_request
+
+      - fail: 
+          msg: PR {{ PR }} does not exist or is closed
+        when: "'closed' in pull_request.json.state"
+      
+      - name: create build tools name space using head/pulls/{{ PR }} as ref 
+        shell: >
+          oc process -f "../templates/react-frontend/bc.yaml" --param-file="../templates/react-frontend/build.param" -p GIT_REF="pull/{{ PR }}/head" -p  IMAGE_TAG="pr-{{ PR }}" | 
```

Comment:
not sure if this is ideal way passing in params from both file and separate ones. Maybe consider turning the build.param file into a template and source it here?

Reply:
what do you mean about that?

Reply:
for example, turn the param file into a jinja template, and make those params (i.e. IMAGE_TAG) optional base on condition. that way you don't need to explicitly set them here.

Reply:
it's good for now, and I think you can proceed with it for sure. templating param values is just nice to have.